doku.php

```diff
@@ -62,7 +62,7 @@
     } else { // check for UNIX Timestamp
         $date_parse = @date('Ymd',$DATE_AT);
         if(!$date_parse || $date_parse === '19700101') {
-            msg(sprintf($lang['unable_to_parse_date'], $DATE_AT));
+            msg(sprintf($lang['unable_to_parse_date'], htmlspecialchars($DATE_AT)));
```

Comment:
That's good. I got the idea from `inc/actions.php`. Will fix that in this pull request as well.